guava-testlib/src/com/google/common/testing/TearDownStack.java

```diff
@@ -74,7 +74,7 @@ public final void runTearDown() {
         tearDown.tearDown();
       } catch (Throwable t) {
         if (suppressThrows) {
-          logger.log(Level.INFO, "exception thrown during tearDown", t);
+          logger.log(Level.SEVERE, "exception thrown during tearDown", t);
```

Comment:
I could see a warning here, but this is a case where the user has explicitly asked this class to suppress exceptions.